classes/WP_CLI/Action/List.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/**
+ * WP-CLI command: action-scheduler action list
+ */
+class ActionScheduler_WPCLI_Action_List_Command extends ActionScheduler_WPCLI_Command {
+
+	const PARAMETERS = array(
```

Comment:
we should mention defaults, especially for pagination params somewhere.

Reply:
```
$ wp help action-scheduler action list
...
SYNOPSIS

  wp action-scheduler action list [--<field>=<value>] [--field=<field>] [--fields=<fields>] [--format=<format>]

  Display actions based on all arguments supported by
  [as_get_scheduled_actions()][1].

  ---
  [1] https://actionscheduler.org/api/#function-reference--as_get_scheduled_actions
...
```

See https://github.com/crstauf/action-scheduler/blob/76e55cfd59282ab2a6a0f2303ff4df0e4dc1469d/classes/WP_CLI/ActionScheduler_WPCLI_Action_Command.php#L209-L213.

This follows the convention set by [the `wp post list` command](https://developer.wordpress.org/cli/commands/post/list/).

Reply:
ok, makes sense. although one change that we should make is change the order to `desc` by default. otherwise I am just seeing list of old actions and it's not immediately helpful.

Reply:
> although one change that we should make is change the order to desc by default.

I will not be making this change: the behavior should match what is documented, and if a user wants to retrieve newer actions, they can use the `date` and `date_compare` parameters.